sparkbox/apprentices-page-next
# feat: hero

## Description

Hero component:
This component contains the logo, intro text/heading and the hero art. 
This PR contains the new hero component and some additions/fixes to the
global and apprentice component style modules.

### Spec

Designs: [Link](https://www.figma.com/file/JcLgjMi1dD3m64Vtfwq0Te/Apprentices?node-id=0%3A1)

See Story: [FSA21V2-142](https://sparkbox.atlassian.net/browse/FSA21V2-142)

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. Browser test with browsers below

## Validation

The following has been completed by the developer:


### Browser Testing

#### Gold Level Browsers

In these browsers, all content is accessible and design matches exactly. In most cases, bugs should be resolved before merge.

**macOS**


**Windows**


**Mobile**


#### Silver Level Browsers

In these browsers, all content is readable, but design may be inaccurate.

**Windows**


**Mobile**


#### Bronze Level Browsers

In these browsers, our only goal is content readability.

**Windows**
